vcd/sample_vcd_test_config.json

```diff
@@ -53,7 +53,11 @@
       "//": "Peer definition for a vpn", 
       "peerIp": "192.168.5.1",
       "peerSubnetGw": "192.168.5.1"
-    }
+    },
+    "vcenter" : "vC1",
+    "externalNetwork": "externalNetworkName",
```

Comment:
"externalNetwork" is used twice in the same scope.

Reply:
Updated